src/pages/ProductDetail/Component/GoToTop.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faChevronUp } from '@fortawesome/free-solid-svg-icons'
+import './GoToTop.scss'
+
+const GoToTop = () => {
+  return (
+    <div className="GoToTop">
```

Comment:
```suggestion
    <div className="goToTop">
```
컴포넌트에서 가장 부모 태그의 이름은 소문자로 시작하는 컴포넌트의 이름이어야 합니다.